tests/FixtureBuilder/ExpressionLanguage/Lexer/SubPatternsLexerTest.php

```diff
@@ -48,6 +48,19 @@ public function testLexAValueToReturnAListOfTokens()
         $this->assertEquals($expected, $actual);
     }
 
+    public function testLexAFunctionContainingLineBreaks()
+    {
+        $expected = [
+            new Token('<identity("foo'.PHP_EOL.'bar")>', new TokenType(TokenType::FUNCTION_TYPE)),
```

Comment:
I think it would be better to use `\n` in one case and `\r\n` for Windows

Reply:
Alright, to be clear does that mean I should do 2 separate test functions for both `\n` and `\r\n` case ? Should I check the OS to make sure it's Windows before running the `\r\n` case ? Also it seems that mac has `\r` line breaks should I update the regex to this : https://stackoverflow.com/a/1331840 ?

Reply:
I think you got an answer regarding the regex on StackOverflow.

You can either go with 2 separate test functions or only one with a data provider, I'm fine with either